hack/Makefile

```diff
@@ -1,21 +1,33 @@
-lint-swagger:
+lint-spectral:
 	spectral lint swagger.yaml
 
+goswagger-validate:
+	goswagger validate swagger.yaml
+
 generate-go-client:
 	rm -rf client/ models/*.go
-	goswagger generate client --template=stratoscale -f swagger.yaml
+	goswagger generate client --skip-validation --template=stratoscale -f swagger.yaml
 
 generate-go-server:
 	rm -rf restapi/
-	goswagger generate server --template=stratoscale -f swagger.yaml
+	goswagger generate server --skip-validation --template=stratoscale -f swagger.yaml
 
 generate-python-client:
 	mkdir -p ${BUILD_FOLDER}
 	./hack/generate.sh generate_python_client
 
-generate-mocks:
+packages-for-generate = $(addsuffix -go-generate-package, $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi'))
```

Comment:
will it work on submodels? 
I know that right now we don't have anything to mock there but if it will be in the future then no one will catch it.
`go list ./...` will skip `api` directory because it contain go.mod

Reply:
This was already the case, I didn't change anything in that regard